deploy/crds/openebs_v1alpha1_openebsinstalltemplate_cr.yaml

```diff
@@ -15,10 +15,10 @@ spec:
     kyvernoEnabled: false
   serviceAccount:
     create: true
-    name: ''
+    name: 'openebs-maya-operator'
```

Comment:
@aamirqs - should we set this or let users continue with what they had in the previous installs.